doc/source/cluster/kubernetes/examples/rayjob-batch-inference-example.md

```diff
@@ -0,0 +1,144 @@
+(kuberay-batch-inference-example)=
+
+# RayJob Batch Inference Example
+
+This page demonstrates how to use the RayJob custom resource to run a batch inference job on a Ray cluster.
+
+We will use an image classification workload.  The example is based on <https://docs.ray.io/en/latest/data/examples/huggingface_vit_batch_prediction.html>. Please see that page for a full explanation of the code.
```

Comment:
What does `<https://docs.ray.io/en/latest/data/examples/huggingface_vit_batch_prediction.html>` look like?

Reply:
Just an ordinary link with the URL spelled out! My linter complains if I don't add "<" and ">"